### DISCUSSION TITLE
java-features/java16/streamToList-vs-collectorsToList

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#9] [Announcements]
# java-features/java16/streamToList-vs-collectorsToList

看到有网友评论问：`Stream.toList()`和`Collectors.toList()`的区别是什么？哪个性能好？今天我们就来测试一下，看看哪个性能更好。

https://www.didispace.com/java-features/java16/streamToList-vs-collectorsToList.html

**Comment by joeljhou** [comment]
hello,你好呀
